src/main/rx/spektrum.h

```diff
@@ -23,4 +23,5 @@
 uint8_t spektrumFrameStatus(void);
 struct rxConfig_s;
 void spektrumBind(struct rxConfig_s *rxConfig);
+extern uint8_t spekRssiEnabled;
```

Comment:
I don't think `spekRssiEnabled` needs to be extern, since it seems local to `rx/spektrum.c`